### ISSUE TITLE
Customize Swagger page with Azure Functions v. 3

### ISSUE THREAD

**Issue by erossini** [yuka1984/azure-functions-extensions-swashbuckle#44]
Great job! I'm using your extension with Azure Functions v. 3 and I don't face any issue. 

![image](https://user-images.githubusercontent.com/9497415/81545468-d6f19c00-9370-11ea-92f2-e9822b94378d.png)

I want to customize the Swagger page, for example the title. I read a [post](https://www.puresourcecode.com/dotnet/csharp/adding-swagger-ui-documentation-to-azure-function-apis/) where it explains how to add some configuration but it doesn't work for me.

Basically, I added some json in the `host.json`

```
{
  "version": "2.0",
  "extensions": {
    ...
 
    "Swashbuckle": {
      "Documents": [
        {
          "Title": "YOUR_TITLE",
          "Version": "v1",
          "Description": "YOUR_DESCRIPTION"
        }
      ]
    }
  }
}
```

What is the right think to do? Thank you!

**Comment by erossini** [comment]
PS: I redeployed the Azure Functions on Azure and now I have an error

```
{
  "schemaValidationMessages": [
    {
      "level": "error",
      "message": "Can't read from file https://azuks-myfnz-q001.azurewebsites.net/api/swagger/json?code="
    }
  ]
}
```

I haven't changed the Startup file. When I published, I deleted existing files

**Comment by vitalybibikov** [comment]
Hi @erossini 
guess it is supported only here at the moment. (v3 functions )
As this repo is abandoned

https://github.com/vitalybibikov/azure-functions-extensions-swashbuckle

You will find all the docs there
